### PR TITLE
LBSD-1238 Syndication switch

### PR DESCRIPTION
**Pull request by m0g**

### CODE REVIEW

**Comment by mscam** [comment]
:+1:

File: client/tasks/options/template.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = function(grunt) {
                 key: grunt.option('embedly-key') || process.env.EMBEDLY_KEY || ''
             },
             facebookAppId: grunt.option('facebook-appid') || process.env.FACEBOOK_APP_ID || '',
+            syndication: true,
```
Comment (nistormihai) [inline client/tasks/options/template.js]: can we set it to `false` by default for now?

File: client/app/index.html [hunk 2]
```diff
@@ -252,6 +251,9 @@ <h3 translate>About</h3>
             'ngMessages'
         ];
 
+        if (config.syndication)
```
Comment (nistormihai) [inline client/app/index.html]: the best way to enable or disable it :+1:

File: client/tasks/options/template.js (hunk 1 above)
Comment (m0g) [inline client/tasks/options/template.js]: oh sure, I'll do directly in https://github.com/liveblog/liveblog/pull/588

**Pull request merged**